templates/server/src/test/resources/projects/basic/archetype.properties

```diff
@@ -5,4 +5,5 @@ groupId=archetype.it
 artifactId=basic
 earProjectName=.
 batch=.
-dbType=h2
\ No newline at end of file
+dbType=h2
+dbMigration=flyway
```

Comment:
should we also have a test using liquibase to ensure the result builds successfully?

Reply:
To do ensure I have added switches here too for liquibase as well as for flyway in all the three archetype properties basic/archetype.properties, batch/archetype.properties and enterprise/archetype.properties.